src/main/streamer/streamerNoHttp.ts

```diff
@@ -266,7 +276,33 @@ const streamProtocolHandler = async (
     // tslint:disable-next-line:max-line-length
     headers["Access-Control-Expose-Headers"] = "Content-Type, Content-Length, Accept-Ranges, Content-Range, Range, Link, Transfer-Encoding, X-Requested-With, Authorization, Accept, Origin, User-Agent, DNT, Cache-Control, Keep-Alive, If-Modified-Since";
 
-    if (isPublicationAssets || isMediaOverlays) {
+    if (isPdfjsAssets) {
+
+        const pdfjsUrlPathname = uPathname.substr(pdfjsAssetsPrefix.length);
+        debug("PDFJS request this file:", pdfjsUrlPathname);
+
+        const pdfjsFolder = "assets/lib/pdfjs";
+        let folderPath: string = path.join(__dirname, pdfjsFolder);
+        if (_PACKAGING === "0") {
+            folderPath = path.join(process.cwd(), "dist", pdfjsFolder);
+        }
+        const pdfjsFullPathname = path.normalize(`${folderPath}/${pdfjsUrlPathname}`);
+        debug("PDFJS full path name :", pdfjsFullPathname);
+
+        const contentLength = `${fs.statSync(pdfjsFullPathname)?.size || 0}`;
+        headers["Content-Length"] = contentLength;
+        const contentType = `${mimeTypes[path.extname(pdfjsFullPathname).slice(1) as keyof typeof mimeTypes] || ""}; charset=utf-8`;
```

Comment:
This will break with non-lowercase extensions (i.e. full upper or mixed case). Please use `findMimeTypeWithExtension()` instead.

Reply:
good catch, thanks you

Reply:
8bd1c77db124ddad425c367f9f68dca949d2435c